R/pin-read-write.R

```diff
@@ -251,6 +256,11 @@ check_metadata <- function(x) {
     abort("`metadata` must be a list")
   }
 }
+check_tags <- function(x) {
+  if (!is.null(x) && !is_character(x)) {
+    abort("`tags` must be a character vector")
```

Comment:
```suggestion
    abort("`tags` must be a character vector.")
```

Ideally you'd say what it actually is too (using `friendly_type`), but if you start going to far down that route you might as well bundle it into a thorough review of argument checking and then use the latest helpers from rlang.